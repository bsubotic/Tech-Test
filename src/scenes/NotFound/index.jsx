import React from "react";
import T from "i18n-react";
import { Link } from "react-router-dom";

import Error from "components/Error";
import routes from "routes";

export default () => (
  <Error>
    <div className="f4 pa3">{T.translate("cantFindPage")}</div>
    <Link to={routes.main.path} className="f6 no-underline pa3 blue-light">
      {T.translate("goToHomepage")}
    </Link>
  </Error>
);
