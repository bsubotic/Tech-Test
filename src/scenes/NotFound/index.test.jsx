import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";

import NotFound from "./index";

test("render an enabled submit button", () => {
  const tree = renderer
    .create(
      <Router>
        <NotFound />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
