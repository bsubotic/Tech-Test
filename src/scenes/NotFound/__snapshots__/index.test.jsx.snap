// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render an enabled submit button 1`] = `
<div
  className="tc"
>
  <svg
    alt="companyName"
    className="w4 pa3"
  >
    acme-logo.svg
  </svg>
  <div
    className="f1 pa4"
  >
    whoops
  </div>
  <svg
    alt="whoops"
  >
    whoops.svg
  </svg>
  <div
    className="f4 pa3"
  >
    cantFindPage
  </div>
  <a
    className="f6 no-underline pa3 blue-light"
    href="/"
    onClick={[Function]}
  >
    goToHomepage
  </a>
</div>
`;
